tests/test_get_vin_history.py

```diff
@@ -81,3 +81,22 @@ def test_vin_non_validate(self):
         vin = '2FMGK5D81EBD1433'
         response = client.get_vin_history(vin)
         self.assertEqual(response, 'Vin is not valid: {}'.format(vin))
+        
+        
+    def test_get_vin_history_with_pagination(self):
+        
+        token = os.getenv('API_TOKEN')
+        client = MarketAPI(token)
+
+        vins = {"1FTEW1EF1FFA67753":6,}
+
+        for vin,page_limit in vins.items():
+            for count in range(1,page_limit+1):
+                time.sleep(3)
+                api_response = client.get_vin_history(vin=vin, page=count)
```

Comment:
`page` keyword arg is not supported by the api client.

Reply:
I use this as a reference for page keyword: 
![image](https://user-images.githubusercontent.com/55404987/73668412-a510cf80-46c7-11ea-9689-60bb34e2686f.png)

https://apidocs.marketcheck.com/?version=latest